java/ql/src/experimental/Security/CWE/CWE-400/LocalThreadResourceAbuse.ql

```diff
@@ -0,0 +1,111 @@
+/**
+ * @name Uncontrolled thread resource consumption from local input source
+ * @description Use user input directly to control thread sleep time could lead to performance problems
```

Comment:
```suggestion
 * @description Using user input directly to control a thread's sleep time could lead to performance problems
```

Reply:
Thanks @atorralba for reviewing this PR again. I've made the text change.